shell/models/fleet.cattle.io.bundle.js

```diff
@@ -30,7 +30,9 @@ export default class FleetBundle extends SteveModel {
   }
 
   get repoName() {
-    return this.metadata.labels['fleet.cattle.io/repo-name'];
+    const labels = this.metadata?.labels || {};
```

Comment:
This seems unrelated but it was throwing an exception on my side.
While possibly uncommon, nothing really prevents using Fleet without a repository. In fact, Fleet itselfs creates bundles this way for the fleet agent